src/AnnotationService.js

```diff
@@ -256,9 +256,9 @@ class AnnotationService extends EventEmitter {
      * @return {Promise} Promise that resolves with fetched annotations
      */
     getReadUrl(fileVersionId, marker = null, limit = null) {
-        let apiUrl = `${this.api}/2.0/files/${this.fileId}/annotations?version=${
-            fileVersionId
-        }&fields=item,thread,details,message,created_by,created_at,modified_at,permissions`;
+        let apiUrl = `${this.api}/2.0/files/${
```

Comment:
Can we extract any substrings to store into constants for readability?

Reply:
I think prettier modified this file in this commit. I'll do that in a separate commit :)